persistence/src/main/java/org/mskcc/cmo/metadb/persistence/PatientMetadataRepository.java

```diff
@@ -0,0 +1,29 @@
+package org.mskcc.cmo.metadb.persistence;
+
+import org.mskcc.cmo.messaging.model.PatientMetadata;
+
+import org.springframework.data.neo4j.annotation.Query;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+import org.springframework.data.repository.query.Param;
+
+/**
+ *
+ * @author ochoaa
+ */
+public interface PatientMetadataRepository extends Neo4jRepository<PatientMetadata, Long> {
+    @Query("MATCH (pm:PatientMetadata) WHERE $investigatorPatientId = pm.investigatorPatientId RETURN pm")
```

Comment:
We should discuss this -  I'm not sure the utility of InvestigatorID - at the most fundamental level, I think that the cmo_metadb_patient_id (uuid) should be used as a primary key for metadata searching.   When a patient node is created in the graphdb, anytime we share patient metadata, we should be including the uuid.

Reply:
this method was added as more of a placeholder/to help me manually test some things